src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles !== '')
```

Comment:
[CODE KNOWLEDGE] - Don't need to make copy before each operation. The `filter` method is being used correctly here as it does not mutate the original array but returns a new one with non-empty styles.